.travis.yml

```
@@ -57,6 +58,7 @@ after_deploy:
   - rm -fr .dat .well-known
   - make-relative https://ournetworks.ca
   - ipd . -O -C
+  - bash scripts/txt.sh ournetworks.ca ipfs-website "ipfs/<hash>"
```

```suggestion
  - bash scripts/txt.sh ournetworks.ca ipfs-website "dnslink=/ipfs/<hash>"
```
ref: http://dnslink.io/

---

Would it make sense to change the `TXT` hostname to `_dnslink.ournetworks.ca`? Following what's on dnslink.io